omod/src/test/java/org/openmrs/module/idgen/web/controller/IdentifierSourceRestControllerTest.java

```diff
@@ -164,10 +163,10 @@ public void shouldUploadIdentifiersFromFile() throws Exception {
         SimpleObject identifiersAfterUpload = deserialize(handle(getRequestAfterUpload));
         assertThat(
                 identifiersAfterUpload.get("identifiers").toString(), allOf(
-                containsString("Test Identifier Pool: 1"),
-                containsString("Test Identifier Pool: 2"),
```

Comment:
Why did you change these?

Reply:
It's because of this change (https://github.com/openmrs/openmrs-module-idgen/pull/74/files#diff-37ab2fee2e4e68dcf200614c09c4a51dR24 -PooledIdentifier). I removed the the IdentifierPool from PooledIdentifier model since its a cyclic dependency and could cause serialization issues with Jackson.

Reply:
I still do not understand how that makes it change from "Test Identifier Pool: 2" to "2"

Reply:
Since I removed the IdentifierPool obviously its name property won't be included in the String we are asserting.

Still curious? See [this](https://github.com/openmrs/openmrs-module-idgen/blob/master/omod/src/test/java/org/openmrs/module/idgen/web/controller/IdentifierSourceRestControllerTest.java#L162), references [this](https://github.com/openmrs/openmrs-module-idgen/blob/master/omod/src/test/java/org/openmrs/module/idgen/web/controller/IdentifierSourceRestControllerTest.java#L43) which later references [this](https://github.com/openmrs/openmrs-module-idgen/blob/master/api/src/test/resources/org/openmrs/module/idgen/include/TestData.xml#L5) - I believe you have seen the name property "Test Identifier Pool"!